src/lib/parse/forbiddenChars.cpp

```diff
@@ -61,3 +61,36 @@ bool forbiddenChars(const char* s)
 
   return false;
 }
+
+
+
+/* ****************************************************************************
+*
+* forbiddenCharsInUriParam - 
+*/
+bool forbiddenCharsInUriParam(const char* s)
```

Comment:
The equal sign '=' is removed also. forbiddenCharsInUriParamWithoutSemicolonAndEqualSign? :-D

I would simply defer this to the new issue #1255 ...
With that issue implemented this function will be removed.